Dockerfile

```diff
@@ -1,6 +1,33 @@
-FROM python:3.9.13-alpine3.16
+FROM ghcr.io/kbase/cdm-spark-standalone:pr-28
 
-# Install Base Packages
-RUN apk --no-cache add bash curl 
+# Switch to root to install packages
+USER root
 
-ENTRYPOINT ["sleep 1d"]
+RUN apt-get update && apt-get install -y \
+    libpq-dev gcc \
```

Comment:
Why do you need these? Could you add a comment?

Reply:
yea. Added a comment. It was be required for the next PR.